PWGCF/FemtoUnited/Tasks/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019-2025 CERN and copyright holders of ALICE O2.
+# See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+# All rights not expressly granted are reserved.
+#
+# This software is distributed under the terms of the GNU General Public
+# License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+#
+# In applying this license CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+o2physics_add_dpl_workflow(femtounited-track-qa
+	  SOURCES femtounitedTrackQa.cxx
+          PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
+          COMPONENT_NAME Analysis)
+
+o2physics_add_dpl_workflow(femtounited-twotrackresonance-qa
+	  SOURCES femtounitedTwotrackresonanceQa.cxx
+          PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
+          COMPONENT_NAME Analysis)
+
+o2physics_add_dpl_workflow(femtounited-v0-qa
+	  SOURCES femtounitedV0Qa.cxx
+          PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
+          COMPONENT_NAME Analysis)
+
+o2physics_add_dpl_workflow(femtounited-cascade-qa
+	  SOURCES femtounitedCascadeQa.cxx
+          PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
+          COMPONENT_NAME Analysis)
+
+o2physics_add_dpl_workflow(femtounited-pair-track-track
+	  SOURCES femtounitedPairTrackTrack.cxx
+          PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
+          COMPONENT_NAME Analysis)
+
+o2physics_add_dpl_workflow(femtounited-pair-track-v0
+	  SOURCES femtounitedPairTrackV0.cxx
+          PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
+          COMPONENT_NAME Analysis)
```

Comment:
In the aim of my previous comments, use just `femto` without the `united`
The QA task are appropriate for this directory
The `pair` tracks I would consider them as example task on how to use the `femto` datamodel
If you aim for this task being your analysis tasks I would recommend leave them as they are in here but copy them and evolve them in the `FemtoDream` directory